packages/lib-classifier/src/plugins/drawingTools/experimental/components/FreehandLine/FreehandLine.js

```diff
@@ -136,6 +137,7 @@ function FreehandLine({ active, mark, onFinish, scale }) {
               strokeWidth: GRAB_STROKE_WIDTH / scale
```

Comment:
Should `GRAB_STROKE_WIDTH` still be divided by scale in this file?

Reply:
The stroke width of each mark on a Correct-a-Cell project is much thinner on [this branch](https://local.zooniverse.org:3000/projects/willcharlie/etch-a-cell-correct-a-cell/classify/workflow/24998?env=production&demo=true) than [production](https://www.zooniverse.org/projects/willcharlie/etch-a-cell-correct-a-cell/classify/workflow/24998).

Reply:
All stroke widths should be constant on this branch.

It’s been a while since I looked at the drawing code, but freehand lines might override the default stroke width, which is quite thick.

Reply:
Visible stroke width for freehand lines is 1px on this branch, but maybe it should be 2px, or whatever the default stroke width is.

Reply:
I've changed STROKE_WIDTH to 2px for freehand lines. I'm not sure why it was set to 1, when other tools use 2. Easy enough to change back, though.

Reply:
I’m not sure why freehand lines override the stroke width that’s set in the base `Mark` component. I don’t think any of the other drawing tools do that.

Reply:
With the latest updates, this PR changes the stroke width of freehand lines. See Correct-a-cell [production](https://www.zooniverse.org/projects/willcharlie/etch-a-cell-correct-a-cell/classify) vs this branch run [locally](https://local.zooniverse.org:3000/projects/willcharlie/etch-a-cell-correct-a-cell/classify). It would be nice to use this PR to improve SVG scaling, but there can't be a noticeable difference when the Correct-a-cell research team views their subjects in the classifier after this PR is merged. (Unless specifically requested by the researchers themselves)

 Please double check all drawing tools on this branch in comparison to production.

Reply:
Have you got any suggestions as to what the width should be? The code says 2px in the Mark component, but 1px in the FreehandLine component.

Reply:
Those Correct-A-Cell lines are styled with `stroke-width: 6.92px`! 😳 

<img width="800" alt="Correct-A-Cell with browser dev tools open to show the CSS styles for an SVG path in a freehand line." src="https://github.com/user-attachments/assets/fa5b0328-5176-4be6-87cf-14f2e17bd902">

Reply:
Compare that to the original Etch-A-Cell on the `production-release` tag, where the line is only a few pixels wide. A big problem with the `scale` factor in the original code is that it produces inconsistent styling from one workflow to another.

https://frontend.preview.zooniverse.org/projects/h-spiers/etch-a-cell/classify/workflow/8921

<img width="800" alt="The original Etch-A-Cell, with browser dev tools open to show the styles applied to SVG freehand lines." src="https://github.com/user-attachments/assets/3fc2043c-6e6a-406e-b1ad-1a7294007568">

Reply:
Some links for testing:
- Etch-A-Cell: https://localhost:8080/?project=h-spiers%2Fetch-a-cell&env=production&workflow=8921
- Correct-A-Cell: https://localhost:8080/?project=willcharlie/etch-a-cell-correct-a-cell&env=production&subject=93925884

When you compare those in production, you can see that Correct-A-Cell lines are thicker than Etch-A-Cell lines. They should be the same on this branch, and Etch-A-Cell freehand lines should be the same width here and in production (I think.)

Reply:
Here's a couple of different-sized Correct-A-Cell subjects. The 1200px subject will be the same size on this branch and in production. The 600px subject will have a thicker line in production, because the `scale` multiplier makes it twice as large. Both subjects should have the same thickness line on this branch, where image scale is ignored.

- 1200px subject: https://localhost:8080/?project=willcharlie/etch-a-cell-correct-a-cell&env=production&subject=93925784
- 600px subject: https://localhost:8080/?project=willcharlie/etch-a-cell-correct-a-cell&env=production&subject=93925884

Reply:
That's going to introduce a complication into the analysis of their beta classifications, since the line width wasn't constant from subject to subject. 😕

Reply:
@goplayoutside3 the stroke width in production depends on both the subject image size (`naturalWidth`) and the rendered image size in the browser (`clientWidth`.) If you want me to match production styles, you’ll need to specify the image width and the subject viewer width, so I know what image scale to aim for. Roughly speaking, the production stroke width in SVG pixels is `2 * naturalWidth / clientWidth` for all the drawing tools except freehand lines. Freehand lines use a scaling function that generates thicker lines for smaller subjects. See https://github.com/zooniverse/front-end-monorepo/issues/6167#issuecomment-2226968539